### DISCUSSION TITLE
please tell me your favorite applications!

### DISCUSSION THREAD

**Discussion by szryo** [NAIST-SE/AttractingContributors#206] [Q&A]
https://github.com/szryo/AttractingContributors

**Comment by naistclass** [comment]
I have sent a Pull Request. Please check.
And will you help me?

https://github.com/naistclass/AttractingContributors-FavoriteFood

**Comment by TakumiHiraide** [comment]
I have added your repository!!
Please help me!!
https://github.com/TakumiHiraide/My-favorite-country

**Comment by hernessteve** [comment]
Oncor, our utility company. It is so much easier to do everything in an app than calling [oncor phone number](https://oncor.pissedconsumer.com/customer-service.html). It takes to much efforts to do it and at the end you still need to submit lots of details via app.

**Comment by ellielo** [comment]
Pocketfied is an awesome app. Here you can find more info about it [https://kukapp.com](url). You can create a mobile app for your online store just in one day. Sounds a bit unreal -right? But it is actually true.  It also does all the updates automatically and adds all the new products as soon as they are available on the website.
